.github/CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
+x. Enhancement - Added freetext category for major HEMS sites and hospitals - thanks to @JamesH2432
```

Comment:
Most contributors add their names to the end e.g. "thanks to @GeekPro101 (Thomas Mills)", but if you'd prefer not to that's absolutely fine